CodeTrans/kubernetes/helm/README.md

```diff
@@ -16,3 +16,150 @@ helm install codetrans oci://ghcr.io/opea-project/charts/codetrans  --set global
 export HFTOKEN="insert-your-huggingface-token-here"
 helm install codetrans oci://ghcr.io/opea-project/charts/codetrans  --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} -f gaudi-values.yaml
 ```
+
+## Deploy on AMD ROCm using Helm charts from the binary Helm repository
+
+```bash
+mkdir ~/codetrans-k8s-install && cd ~/codetrans-k8s-install
+```
+
+### Cloning repos
+
+```bash
+git clone https://github.com/opea-project/GenAIExamples.git
+```
+
+### Go to the installation directory
+
+```bash
+cd GenAIExamples/CodeTrans/kubernetes/helm
+```
+
+### Settings system variables
+
+```bash
+export HFTOKEN="your_huggingface_token"
+export MODELDIR="/mnt/opea-models"
+export MODELNAME="mistralai/Mistral-7B-Instruct-v0.3"
+```
+
+### Setting variables in Values files
+
+#### If ROCm vLLM used
+```bash
+nano ~/codetrans-k8s-install/GenAIExamples/CodeTrans/kubernetes/helm/rocm-values.yaml
+```
+
+- HIP_VISIBLE_DEVICES - this variable specifies the ID of the GPU that you want to use.
+  You can specify either one or several comma-separated ones - "0" or "0,1,2,3"
+- TENSOR_PARALLEL_SIZE - must match the number of GPUs used
+- resources:
+  limits:
+  amd.com/gpu: "1" - replace "1" with the number of GPUs used
+
+#### If ROCm TGI used
+
+```bash
+nano ~/codetrans-k8s-install/GenAIExamples/CodeTrans/kubernetes/helm/rocm-tgi-values.yaml
+```
+
+- HIP_VISIBLE_DEVICES - this variable specifies the ID of the GPU that you want to use.
+  You can specify either one or several comma-separated ones - "0" or "0,1,2,3"
+- extraCmdArgs: [ "--num-shard","1" ] - replace "1" with the number of GPUs used
+- resources:
+  limits:
+  amd.com/gpu: "1" - replace "1" with the number of GPUs used
+
+### Installing the Helm Chart
+
+#### If ROCm vLLM used
+```bash
+helm upgrade --install codetrans oci://ghcr.io/opea-project/charts/codetrans \
+    --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} \
+    --values rocm-values.yaml
+```
+
+#### If ROCm TGI used
+```bash
+helm upgrade --install codetrans oci://ghcr.io/opea-project/charts/codetrans \
+    --set global.HUGGINGFACEHUB_API_TOKEN=${HFTOKEN} \
+    --values rocm-tgi-values.yaml
+```
+
+## Deploy on AMD ROCm using Helm charts from Git repositories
+
+### Creating working dirs
+
+```bash
+mkdir ~/codetrans-k8s-install && cd ~/codetrans-k8s-install
```

Comment:
Once you created the directory, cd to it. All the other paths then get shorter, do not need to reference ~/codetrans-k8s-install everywhere